### PR TITLE
Linux Mint 22 support

### PR DESCRIPTION
**Pull request by jeremiah-k**
Linux Mint 22 was released today. 

With these changes I was able to install zerotier-one successfully. Also a few minor typo fixes.

```
NAME="Linux Mint"
VERSION="22 (Wilma)"
ID=linuxmint
ID_LIKE="ubuntu debian"
PRETTY_NAME="Linux Mint 22"
VERSION_ID="22"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.linuxmint.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=wilma
UBUNTU_CODENAME=noble
```

### CODE REVIEW

**Comment by jeremiah-k** [comment]
I tried to squash commits and I don't have much experience with it.  I apologize if this is sloppy.

**Review by Squelch — APPROVED** [review]
The changes correctly map Linux Mint 22 Wilma to Ubuntu Noble. This should enable successful installation on this newly released LTS

Due to the malformed apt cache, software installation and updates are disabled, so this fix should have a high priority for merging and release.

**Review by laduke — APPROVED** [review]

**Review by laduke — DISMISSED** [review]

File: htdocs/install.sh [hunk 1]
```diff
@@ -361,15 +361,15 @@ elif [ $ID == "opensuse" ] || [ $ID == "suse" ]; then
 	cat /dev/null | $SUDO rpm --import /tmp/zt-gpg-key
 
 	echo
-	echo '*** Installing zerotier-one package...'
```
Comment (Squelch) [inline htdocs/install.sh]: This change reverts spelling correction

File: htdocs/install.sh [hunk 2]
```diff
@@ -361,15 +361,15 @@ elif [ $ID == "opensuse" ] || [ $ID == "suse" ]; then
...
+	echo '*** Installing zeortier-one package...'
 
 	cat /dev/null | $SUDO zypper install -y zerotier-one
 elif  [ $ID == "opensuse-tumbleweed" ]; then 
 	echo '*** Found SuSE Tumbleweed/Leap, adding zypper YUM repo...'	
 	cat /dev/null | $SUDO zypper addrepo -t YUM -G ${ZT_BASE_URL_HTTP}redhat/el/9 zerotier
 
 	echo
-	echo '*** Installing zerotier-one package...'
```
Comment (Squelch) [inline htdocs/install.sh]: This change reverts spelling correction

**Review by Squelch — CHANGES_REQUESTED** [review]
Two of the spelling corrections were reverted during signing.
Both annotated.

Note: These are benign typos, but are shown in console output.

File: htdocs/install.sh (hunk 1 above)
Comment (glimberg) [reply htdocs/install.sh]: looks like you edited htdocs/install.sh.  Only  /install.sh.in should be edited

Comment (jeremiah-k) [reply htdocs/install.sh]: Ah yeah, I wasn't sure of the relationship between those and see the build script now. I reverted the changes to htdocs/install.sh

**Review by laduke — APPROVED** [review]
thanks

**Comment by glimberg** [comment]
will get this out shortly

File: htdocs/install.sh (hunk 1 above)
Comment (Squelch) [reply htdocs/install.sh]: @glimberg Does that imply that /htdocs/install.sh should be a linked file to /install.sh.in and excluded via git config?

Comment (glimberg) [reply htdocs/install.sh]: no not at all.  `htdocs/install.sh` is a copy of `install.sh.in` that includes the GPG signature of the content

**Pull request merged**